### DISCUSSION TITLE
No more Update Information

### DISCUSSION THREAD

**Discussion by Taste-IT** [tine20/tine20#7357] [General]
I noticed, that i get no more information about newer version than i have installed. I think this is since last year. Does anything change, or i have i enable it now?
thx

**Comment by pschuele** [comment]
hi @Taste-IT ,
we changed our release schedule to weekly releases and currently do not update our "new version is available" API. maybe we only update it when we have our yearly "upgrade" release.

see https://github.com/tine20/tine20/issues/7351 for more information.

**Comment by Taste-IT** [comment]
Hi,
what is the reason behind that? I think its an automatism of asking github for a new version, or not? Because notification is very important. Am I right and if i see a new update like the present 2022.05.21.91, this is the weekly one? Its an nightly right? Normaly i take the latest stable from packages.tine20.org/sources, but these seems to be very old. Where can i find the latest stable?
thx

**Reply by texiuozelotl** [reply]
Ich mach mal auf deutsch weiter: Ich nutze Tine seit der Absplitterung von eGroupware für mein kleines Gewerbe und als Familien-Cloud und war lange Zeit sehr zufrieden damit. Mittlerweile schlage ich mich aber auch mit dem Gedanke herum komplett auf Nextcloud zu switchen. Nachdem zuerst Forum und Community-Website abgeschaltet wurden, gibt's nun auch keine fertig gebauten ZIP-Releases mehr, brauchbare Dokumentation ebenfalls Fehlanzeige. Bitte versteht mich nicht falsch, das ist kein Vorwurf - ich bringe nur mein Bedauern zum Ausdruck, dass es diesem tollen Projekt nicht gelungen ist die nötigen (Community-) Ressourcen zu akkumulieren um zumindest die Basics zu vermitteln / dokumentieren, wie man die Software nun installiert oder geupdated bekommt. Ich persönlich bin seit etlichen Jahren nicht mehr in der aktiven Webentwicklung tätig und habe keinen Plan mehr, wie ich Tine jetzt noch aktualisiert bekomme. Vielleicht mag mir mal jemand eine Tine-for-Dummies Anleitung schreiben - darüber würde ich mich sehr freuen :-) Denn eigentlich möchte ich wirklich nicht zu Nextcloud wechseln: Die machen zwar schönes File, Sync & Share, aber wenn ich deren Kalender, Adressbuch und Mail-App ansehe, dann will ich das nun wirklich nicht gegen die Tine-Features eintauschen.

LG
Martin

**Reply by rwkstgtd** [reply]
Ich bin mittlerweile zu Nextcloud gewechselt, weil ich zu dumm oder die Update-Anleitung zu schlecht für Leute mit mittelmäßigen web/php-Kenntnissen ist. Auch war mir das Chaos mit den nightly-builds zu viel. Ich hätte mit ein paar Stunden Arbeit sicher auch das nächste Tine-Update hinbekommen, aber die habe ich nicht immer wieder und wieder und wieder. Ich hatte ein Skript für die automatische Aktualisierung, aber das hätte dank des Fehlens von Community-Releases jetzt auch wieder angepasst werden müssen. Aufwand steht in keinem Verhältnis zum Ergebnis.
Ich habe bisher nur den Kalender genutzt (Familie, Einzelunternehmer) und der funktioniert bei Nextcloud deutlich besser (endlich kann man unter Android verschiedene Kalender, die einer Person freigegeben wurden, unterscheiden, bisher war das alles eins; ein Termin im Gruppenkalender sah aus, wie einer im eigenen Kalender).
Cool ist bei Nextcloud das automatisierte Update, die Oberfläche wirkt auch weniger angestaubt und lädt flüssiger, als bei Tine. Die besseren Möglichkeiten, Dateien zu teilen, sind für mich nützlich. Die Mail-App benötige ich nicht, sie mag schlecht sein, weiß ich nicht.
Will nur sagen: Ich würde Nextcloud zumindest mal testen. Mit etwas Anstrengung habe ich es auch geschafft, alle Termine der letzten 10 Jahre zu migrieren. Anstrengung auf der Tine-Seite, Import in Nextcloud war sehr einfach.

**Reply by pschuele** [reply]
hi,
we only build "stable" upgrade releases once a year - you can find those (again) on https://packages.tine20.org/source/.
the 2022 yearly release (2022.12.1) has just been published there.

the weekly releases will still be coming each Saturday. maybe it's a good idea to only notify for the "stable" (or important) security releases? I'll think about that.

I also updated the wiki update page to reflect the recent changes: https://github.com/tine20/tine20/wiki/EN:Update-Howto.

if you struggle with deploying the updates, maybe the docker version is the thing for you (get it from https://hub.docker.com/r/tinegroupware/tine).

**Reply by lab-at-nohl** [reply]
Please do not notify nightlies.

**Comment by agent-ck** [comment]
Ich habe Tine20 durch meinen Arbeitgeber kennen gelernt. Es hat mir damals so gut gefallen, dass ich auch zu Hause eine VM mit Tine20 aufgesetzt und privat für die Familie genutzt habe. Es sollte meinen Exchange 2010 ablösen, da ich keine Lizenzen für neuere Versionen gekauft habe. Durch den Exchange bekam ich Kontakt zu John Bieling (@jobisoft) und richtete auch für ihn ein Konto auf einem Test-Tine ein.

Dadurch, dass ich nun mehrere Tine20 Instanzen zur Verfügung hatte (Arbeit/Privat, Test/Prod), konnte ich eigentlich fast alles automatisieren, inkl. der Updates. Doch Metaways hat mir immer wieder Fallstricke gespannt und meine Automatismen gebrochen, angefangen von abgelaufenenen GPG-Keys, über Ändern von Download-Urls, über den Wegfall von Installationspaketen, bis hin zum heutigen Desaster mit den unklaren Installationsquellen. 

In der Arbeit hatten wir schon lange die Nase voll und entschieden uns zur Migration auf Microsoft 365. Da klappte von Anfang an alles. Die einzigen Probleme waren die User, welche mit den neuen Oberflächen nicht klar kamen, aber auch das hat sich sehr schnell gelegt. Microsoft ist was das betrifft wesentlich ausgereifter und intuitiver.

Privat wollte ich Tine20 noch beibehalten, aber der Update-Mechanismus ist schon lange Geschichte, und selbst die manuellen Updates klappen nicht. Tine20 lief schon sehr lange nicht mehr rund und erst kürzlich habe ich mir die Tine20 Installation durch ein Update komplett unbrauchbar gemacht, weil auf einmal 3-party Komponenten und irgendwelches Developergedöns fehlten.

Mir ist der ganze Aufwand inzwischen zu viel und Tine20 ist es mir nicht mehr wert, dass ich meine Freizeit darin investiere. Ich habe meine VMs dekommissioniert. Tine20 spielt keine Rolle mehr für mich! Mir tut es nur leid für John, aber ich denke, er hat noch genug andere Clients zum Entwickeln.

Manchmal habe ich den Eindruck, dass Metaways die Community Version eigentlich gar nicht mehr haben möchte, weil sie nur Aufwand erzeugt und kein Geld abwirft. Dann aber sage ich mir immer wieder, das ist Quatsch! Tine20 lebt ja auch durch die Community. Heute glaube ich, dass Tine20 einfach zu groß geworden ist und die Entwickler sich selbst die Bürde auferlegt habe, immer mehr in immer kürzerer Zeit rein zu bringen, koste es was es wolle! Darunter leidet natürlich die Qualität und erst recht die Dokumentation. Viel schlimmer ist aber, dass langfristig aber das Produkt einen schlechten Ruf bekommt. 

Mein Tipp an die Verantwortlichen ist Folgender:
Stellt euch doch einfach mal die folgenden Fragen:
1. Was war nochmal der Grund, warum sich Tine von eGroupware abgespalten hat?
2. Begeht ihr langsam vielleicht diesselben Fehler?
3. Wann ist jemand so unzufrieden, dass er einen neuen unabhängigen Fork von Tine macht?

Und dann wäre es schön, wenn wir das Gas wieder raus nehmt und mehr an der Qualität anstatt an der Quantität arbeitet - mit allem was dazu gehört, inkl. vernünftige Dokumentation.

**Comment by jobisoft** [comment]
Vielen Dank für die Info und vielen Dank für deine geleistete Unterstützung. In meinem Umfeld sind auch viele auf O365 gewechselt - einfach, weil es funktioniert. 

Viele alternative Projekte wirken einfach noch immer wie Bastellösungen und benötigen Enthusiasmus und Zeit. Vor ein paar Jahren, als man noch jung war und die Zeit hatte, da ging das (und man konnte ja auch etwas Geld sparen) - aber heute, mit Familie, Arbeit und unzähligen Verpflichtungen, da ist mir zumindest klar geworden, dass Lebenszeit ja auch einen gewissen Wert hat. Man zahlt dann lieber für eine funktionierende Dienstleistung, kann dafür aber seine knappe Zeit für wirklich schöne Dinge nutzen.

**Reply by agent-ck** [reply]
Hallo John (@jobisoft),

leider ist der Exchange inzwischen auch nicht mehr einsatzfähig. Ich kann dich also hier auch nicht mehr unterstützen. Allerdings habe ich noch (bzw. wieder) Resourcen frei. Falls du mal ein Testsystem brauchen solltest, schreib mich an. (Emailadresse(n) hast du hoffentlich noch!?)

**Comment by 2greatsecret** [comment]
Ich habe eine gegenteilige Frage: Wie kann ich die Meldung, dass ein Update verfügbar ist deaktivieren?
Momentan nutze ich die Version "2021.12.1" und bekomme die Meldung, dass "2022.06 2022.06.18.130" verfügbar ist.
Da ich keine Anleitung finde, wie ich auf dem neuen Build System umsteigen kann, möchte ich bei der installierten Version bleiben, jedoch nicht jedes Mal daran erinnert werden, dass es ein Update gibt, das ich so wie so nicht installieren kann...

**Reply by agent-ck** [reply]
Ja, die lieben Versionshinweise ... 

Normalerweise ist das ein Haken in den Rolleneinstellungen (Tinebase), aber selbst das hat bei mir irgendwann nicht mehr funktioniert. An einem Punkt hatte ich die Nase so voll, dass ich die Update-Url gesucht und die Domain in die /etc/hosts auf 127.0.0.1 umgebogen hatte.

Genauer kann ich es dir nicht schreiben, da ich kein Tine mehr installiert habe und auch nicht mehr installieren werde, es sei denn, die Dokumentation wird wieder vernünftig geführt.

**Comment by pschuele** [comment]
danke für euer feedback - wir werden uns dazu austauschen und hoffentlich eine lösung finden, wie wir den update-prozess in der community edition zukünftig einfacher gestalten können.
